tools/db_crashtest.py

```diff
@@ -36,6 +36,8 @@
     # Consider larger number when backups considered more stable
     "backup_one_in": 100000,
     "batch_protection_bytes_per_key": lambda: random.choice([0, 8]),
+    # 1 is too flaky
```

Comment:
It doesn't cause false positive right? I think 1/256 chance of missing corruption is fine for crash test.

Reply:
You are right, I confused this with unit tests where false negative could be flaky.